transformer_engine/jax/cpp_extensions/normalization.py

```diff
@@ -239,12 +241,77 @@ def partition(zero_centered_gamma, epsilon, mesh, arg_infos, result_infos):
 register_primitive(LayerNormFwdPrimitive)
 
 
+def _layernorm(x, gamma, beta, zero_centered_gamma, eps):
```

Comment:
If we have all "native jax" implementations for these functions here which are equivalent to reference functions in the unit test, I think we should update unit tests to import and use these functions directly.

Reply:
I would like to maintain two separate implementations for both the exposed APIs and unit tests. Otherwise, the reference for the ‘customcalls disable’ path will be self-referential.